### PR TITLE
handle input packages

### LINKED ISSUE #5: Include fields for type=input packages
No fields are included with input type packages. See the CEL test case.

### PR DESCRIPTION
**Pull request by efd6**
Please take a look.

Fixes #5

### CODE REVIEW

File: fleetpkg.go [hunk 1]
```diff
@@ -455,15 +455,25 @@ func Read(path string) (*Integration, error) {
...
 		}
-		integration.DataStreams[filepath.Base(ds.sourceDir)] = ds
+		if integration.Manifest.Type == "input" {
+			integration.Input = ds
+		} else {
+			integration.DataStreams[filepath.Base(ds.sourceDir)] = ds
+		}
 
 		if err := readYAML(manifestPath, &ds.Manifest, true); err != nil {
 			return nil, err
```
Comment (efd6) [inline fleetpkg.go:479]: Probably want to conditionally omit this and the following couple of lines when we are an input package.

File: testdata/cel.golden.json [hunk 2]
```diff
@@ -362,6 +362,100 @@
       "github": "elastic/security-external-integrations"
     }
   },
+  "input": {
+    "manifest": {},
```
Comment (andrewkroh) [inline testdata/cel.golden.json:366]: I probably should have made the DataStream.Manifest a `*DataStreamManifest` so this could be omitted.

**Review by andrewkroh — APPROVED** [review]
LGTM. Thanks.

**Comment by andrewkroh** [comment]
Released in [v0.0.11](https://github.com/andrewkroh/go-fleetpkg/releases/tag/v0.0.11).

**Pull request merged**